rust/protocol/src/storage/traits.rs

```diff
@@ -1,38 +1,70 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2021 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
+//! Traits defining several stores used throughout the Signal Protocol.
+
+// TODO
+// #![warn(missing_docs)]
```

Comment:
:arrow_up_down:

Reply:
Done! We can actually remove the `#![warn(missing_docs)]` in this module and just set it in the parent module in `storage.rs`!